kmath-geometry/src/commonMain/kotlin/scientifik/kmath/geometry/transformations/Relation.kt

```diff
@@ -0,0 +1,14 @@
+package scientifik.kmath.geometry.transformations
+
+import scientifik.kmath.dimensions.Dimension
+import scientifik.kmath.geometry.InnerProductSpace
+import scientifik.kmath.geometry.Vector
+
+interface Relation<T1, D1 : Dimension, V1 : Vector<T1, D1>, T2, D2 : Dimension, V2 : Vector<T2, D2>> : (V1) -> V2 {
```

Comment:
Should be documented too.

Reply:
Added documentation to classes in `scientifik.kmath.geometry.transformations` package, please check.